packages/amazon-bedrock/src/bedrock-embedding-model.ts

```diff
@@ -27,7 +27,7 @@ type DoEmbedResponse = Awaited<ReturnType<EmbeddingModelV2<string>['doEmbed']>>;
 export class BedrockEmbeddingModel implements EmbeddingModelV2<string> {
   readonly specificationVersion = 'v2';
   readonly provider = 'amazon-bedrock';
-  readonly maxEmbeddingsPerCall = undefined;
+  readonly maxEmbeddingsPerCall = 1;
```

Comment:
doubt that is correct - most embedding calls will take forever with 1 embedding per call. where is this number from?

Reply:
https://github.com/vercel/ai/blob/61020b3a19361781efd8ed36895fcd9a60eb99c7/packages/amazon-bedrock/src/bedrock-embedding-model.ts#L90-L94

Reply:
oh this is interesting - the provider should not do this, what am i missing?

Reply:
created a task in ai sdk v5 doc